deps/build.jl

```diff
@@ -22,45 +24,37 @@ julia_binary = get(ENV, "JULIA_BINARY", Sys.BINDIR)
 ## Install GAP
 if install_gap
     println("Installing GAP ...")
-    gap_branch = "stable-4.11"
     cd(extra_gap_root)
-    run(`rm -rf gap`)
-    run(`git clone --depth=1 -b $(gap_branch) https://github.com/gap-system/gap`)
-    cd("gap")
-    run(`./autogen.sh`)
+    run(`rm -rf gap-$(gap_version)`)
+    filename = download("https://www.gap-system.org/pub/gap/gap-4.11/tar.bz2/gap-$(gap_version).tar.bz2")
```

Comment:
Can we avoid the hard-coded 4.11 in this line (and in etc/travis_install_gap.sh)?